esp/src/eclwatch/LFDetailsWidget.js

```diff
@@ -462,6 +464,12 @@ define([
                 this.updateInput("RecordSize", oldValue, this.i18n.NoPublishedSize);
             } else if (name === "Cost") {
                 this.updateInput("FormattedCost", oldValue, Session.formatCost(newValue));
+            } else if (name === "MinSkew") {
+                var formatNum = hpccCommon.format(",.2f");
+                domAttr.set(this.id + "MinSkew", "innerHTML", this.logicalFile.MinSkew ? formatNum(this.logicalFile.MinSkew) + "%" : "");
+            } else if (name === "MaxSkew") {
+                var formatNum = hpccCommon.format(",.2f");
```

Comment:
formatNum declaration should be moved to module scope or the Util module as "skewFormat"?